src/gui/categoryfilterwidget.cpp

```diff
@@ -73,6 +73,8 @@ CategoryFilterWidget::CategoryFilterWidget(QWidget *parent)
     setIconSize(Utils::Misc::smallIconSize());
 #if defined(Q_OS_MAC)
     setAttribute(Qt::WA_MacShowFocusRect, false);
+    if (!BitTorrent::Session::instance()->isSubcategoriesEnabled())
```

Comment:
But if `Use Subcategories` is changed it would need a restart for the indentation to change. Is that ok?
If you can make it change without restart i would like to suggest to move this out of the `#if` so all platforms can have it.